tests/integration/telemetry/constants.go

```diff
@@ -19,4 +19,8 @@ const (
 	// For multicluster tests we multiply the number of requests with a
 	// constant multiplier to make sure we have cross cluster traffic.
 	RequestCountMultipler = 3
+	// Default retry delay used in tests.
+	RetryDelayInSeconds = 3
```

Comment:
Even better, could we make these flags with the values here as the defaults? That way we can tweak them dynamically if needed.

Reply:
Good idea.. done